src/editors/containers/VideoUploadEditor/VideoUploader.jsx

```diff
@@ -17,49 +16,51 @@ const URLUploader = () => {
   const intl = useIntl();
   return (
     <div className="d-flex flex-column">
-      <div style={{ backgroundColor: '#F2F0EF' }} className="justify-content-center align-self-center rounded-circle p-5">
-        <Icon src={FileUpload} className="text-muted" size="lg" />
+      <div className="justify-content-center align-self-center rounded-circle bg-light-300 p-2.5">
+        <Icon src={FileUpload} className="text-muted" style={{ height: '2rem', width: '2rem' }} />
       </div>
-      <div className="d-flex align-self-center justify-content-center flex-wrap flex-column pt-5">
-        <span className="small">{intl.formatMessage(messages.dropVideoFileHere)}</span>
-        <span className="align-self-center" style={{ fontSize: '0.8rem' }}>{intl.formatMessage(messages.info)}</span>
+      <div className="d-flex align-self-center justify-content-center flex-wrap flex-column pt-3">
+        <span>{intl.formatMessage(messages.dropVideoFileHere)}</span>
+        <span className="x-small align-self-center pt-2">{intl.formatMessage(messages.info)}</span>
       </div>
-      <div className="x-small align-self-center justify-content-center mx-2 text-dark font-weight-normal">OR</div>
-      <div className="zindex-9 video-id-prompt p-4">
-        <InputGroup className="video-upload-input-group">
+      <div className="small align-self-center justify-content-center mx-2 text-dark font-weight-normal pt-3">
+        OR
+      </div>
+      <div className="zindex-9 video-id-prompt py-3">
+        <InputGroup>
           <FormControl
+            className="m-0"
```

Comment:
When the link for a video is extends the full width of the input box, the submit arrow covers the text.

<img width="396" alt="Screenshot 2023-11-30 at 3 38 01 PM" src="https://github.com/openedx/frontend-lib-content-components/assets/42981026/3bee4c67-6853-48ff-97cf-fb1da98d1ccc">